docs/site/Loopback-component-authorization.md

```diff
@@ -8,543 +8,377 @@ permalink: /doc/en/lb4/Loopback-component-authorization.html
 
 ## Overview
 
-In every web application, we need to have a way to identify access rights of a
-user for any resource, which is known as **Authorization**. This is a
-minimalistic guide for creating such an implementation using Loopback component.
-This can be part of your main REST Application project or can be created as a
-Loopback extension for reuse in multiple projects. Latter is the better option
-for obvious reasons - reusability.
-
-{% include note.html content="
-The LoopBack team is working on making authorization an out-of-the-box
-feature in LoopBack 4. It is a work in progress and will soon be there. Until
-then, this implementation guide can be followed.
-" %}
-
-## The requirement
-
-1. Every protected API end point needs to be restricted by specific permissions.
-2. API allows access only if logged in user has permission as per end point
-   restrictions.
-3. API throws **403 Forbidden** error if logged in user do not have sufficient
-   permissions.
-4. Publicly accessible APIs must be accessible regardless of user permissions.
-5. Every user has a set of permissions. These permissions may be associated via
-   role attached to the user or directly to the user.
-6. A user can be provided additional permissions or denied some permissions over
-   an above its role permissions. This is considered explicit allow/deny and
-   always takes precedence while calculating permissions.
-
-## Considerations
-
-There are a few considerations that are taken into account before this
-implementation can be done.
-
-1. User authentication is already implemented. You can refer to the
-   [@loopback/authentication](https://github.com/strongloop/loopback-next/tree/master/packages/authentication#loopbackauthentication)
-   guide.
-2. As part of authentication, client is sent back a token (JWT or similar) which
-   client need to pass in every API request headers thereafter.
-3. The authenticate action provider parses the token to return AuthResponse
-   object.
-4. AuthResponse contains the logged in user information including associated
-   role details.
-
-## The implementation
-
-First, let's define the types needed for this.
-
-{% include code-caption.html content="/src/authorization/types.ts" %}
+Authorization decides if a subject can perform specific action on an object.
```

Comment:
The term `subject` and `object` are used without links or introductions. Is it easy to understand?

Reply:
So, `a subject can perform specific action on an object` is provided as a textbook definition of Authorization. do we need to provide more details on the definition of Authorization ?

Reply:
@raymondfeng I have modified the text now to be more clear